playwright.config.ts

```diff
@@ -33,19 +33,36 @@ export default defineConfig({
 
   /* Configure projects for major browsers */
   projects: [
+    // Setup project
```

Comment:
If we don't need the commented out `setup` stuff, can we remove it?

Reply:
Yes, I will remove the commented out part in the config.